src/vs/workbench/contrib/chat/browser/chatViewPane.ts

```diff
@@ -155,7 +157,9 @@ export class ChatViewPane extends ViewPane implements IViewWelcomeDelegate {
 
 			const scopedInstantiationService = this._register(this.instantiationService.createChild(new ServiceCollection([IContextKeyService, this.scopedContextKeyService])));
 			const locationBasedColors = this.getLocationBasedColors();
-			const editorOverflowNode = $('.chat-editor-overflow-widgets');
+			const editorOverflowNode = this.layoutService.mainContainer.appendChild($('.chat-editor-overflow.monaco-editor'));
```

Comment:
✋ how does this work with floating windows? `mainContainer` is only main window 🤔

Reply:
Good point. In this case though I think the chat view pane can only exist in the main window, right?

Reply:
Chat can be opened as editor, and thus move into floating windows.